bootloader/iot-lab_M3/iotlab-m3.cfg

```diff
@@ -1,16 +1,8 @@
-# Copied from iot-lab/parts/openlab/platform/scripts/iotlab-m3.cfg
-
-adapter_khz 1000
-
-# comstick ftdi device
-interface ft2232
-ft2232_layout "usbjtag"
-ft2232_device_desc "FITECO M3"
-ft2232_vid_pid 0x0403 0x6010
-
-jtag_nsrst_delay 100
-jtag_ntrst_delay 100
+source [find interface/ftdi/iotlab-usb.cfg]
```

Comment:
where is the path interface/ftdi/iotlab-usb.cfg

Reply:
In `openocd` https://github.com/ntfreak/openocd/blob/master/tcl/interface/ftdi/iotlab-usb.cfg

Reply:
This should be found automatically by openocd.